app/src/main/res/values-nl/strings.xml

```diff
@@ -30,7 +30,7 @@
     <string name="ongoing_call">Gesprek gaande</string>
     <string name="select_sim">Selecteer de SIM-kaart voor dit gesprek</string>
     <string name="always_use_this_sim">Voor dit nummer altijd deze SIM-kaart gebruiken</string>
-    <string name="unset_default_sim">Unset default SIM</string>
+    <string name="unset_default_sim">Instelling standaard SIM-kaart verwijderen</string>
```

Comment:
could this be shortened maybe? It is for a top menu item and it is way too long.

Reply:
Yes, but before I change it, is always_use_this_sim also a top menu item?

Reply:
no, that is only at the dialog in the middle of the screen, where you can select a SIM card, and a checkbox with the given label is beneath it. Like at file sorting in some apps + use for this folder only.

Reply:
All right, changed 👍

Reply:
ye looks good, thanks